_layouts/500.html

```diff
@@ -0,0 +1,48 @@
+{% extends "layout-full.html" %}
+
+{% block title -%}
+    500 internal server error
+{%- endblock %}
+
+{% block desc -%}
+
+{%- endblock %}
+
+{% block content_modifiers -%}
+	error_main
+{%- endblock %}
+
+{% block content_main %}
+    <div class="error_copy_container">
+        <section class='error_heading u-mt30'>
+            <h1 class='superheader'>500 internal server error</h1>
+            <p class='short_desc'>We can’t find the page you were
+                looking for, but here are
+                a number of other ways you
+                might try to get there.
+            </p>
+        </section>
+        <aside class='block padded-container u-mt30 u-mb60 '>
```

Comment:
Minor...
- Trailing whitespace.
- You may want to update this file to use double-quotes throughout per https://github.com/cfpb/front-end/blob/master/markup.md#quotes